src/components/ICA/ICA.js

```diff
@@ -81,20 +107,36 @@ const ICA = ({
     numeral.locale(ICA.defaultProps.locale);
   }
 
-  const truncatedValue = truncateValue(percentage, value, truncate);
+  const truncatedValue = truncateValue(percentage, value, truncate, ICAClasses);
   const truncatedTotal = formatValue(total, truncate);
 
   const shouldDisplayof =
     (!percentage && total > value && truncatedValue !== truncatedTotal) ||
     (forceShowTotal && truncatedTotal > 0);
 
   return (
-    <div className={`${namespace} ${className}`} {...other}>
-      <h4 className={`${namespace}__label`}>{label} </h4>
-      <span className={`${namespace}__value`}>{truncatedValue}</span>
-      {shouldDisplayof ? (
-        <span className={`${namespace}__total`}> / {truncatedTotal}</span>
-      ) : null}
+    <div className={ICAClasses} {...other}>
+      <h4 className={`${namespace}__label ${ICAClasses}__label`}>{label} </h4>
```

Comment:
As I understand, with the inclusion of `ICAClasses`, this will output `security--ica__label security--ica security--ica--large__label`, which could cause unintended side effects.

If this is to hook into styles for sizes, would it be sufficient to use a selector like `&--lg &__label`? This could potentially be applied anywhere `ICAClasses` is used.

Reply:
I removed the redundancy in the ICAClasses setup. It should now output `security--ica__label  security--ica--large`. Does this seem like a good fix?

Reply:
Apologies for any confusion so far - we can keep the `icaClasses` on the return function's root node, and remove it everywhere else as the styles can handle it from the top level. For example:

```js
  return (
    <div className={icaClasses} {...other}>
      <h4 className={`${namespace}__label`}>{label} </h4>
      ...
```

and in the styles:

```scss
@include security--component($name: ica) {

  &--lg & {
    &__label {
      ...
    }

    &__value {
      ...
    }
  }

  &--xl & {
    &__label {
      ...  
    }
  }

  ...
}
```